core/editor/src/communication/dispatcher.rs

```diff
@@ -220,8 +221,8 @@ mod test {
 
 		editor.handle_message(Message::Document(DocumentMessage::CopySelectedLayers)).unwrap();
 		editor.handle_message(Message::Document(DocumentMessage::DeleteSelectedLayers)).unwrap();
-		editor.handle_message(Message::Document(DocumentMessage::PasteLayers)).unwrap();
-		editor.handle_message(Message::Document(DocumentMessage::PasteLayers)).unwrap();
+		editor.handle_message(Message::Document(DocumentMessage::PasteLayers { path: vec![], insert_index: -1 })).unwrap();
+		editor.handle_message(Message::Document(DocumentMessage::PasteLayers { path: vec![], insert_index: -1 })).unwrap();
```

Comment:
Why two identical lines?

Reply:
because it is part of the tests

Reply:
Oh, I failed to notice that was in a test. 👍